README.md

```diff
@@ -18,6 +18,7 @@ Keycloak provides user federation, strong authentication, user management, fine-
 
 * [Documentation](https://www.keycloak.org/documentation.html)
 * [User Mailing List](https://groups.google.com/d/forum/keycloak-user) - Mailing list for help and general questions about Keycloak
+* Join [#keycloak](https://cloud-native.slack.com/archives/C056HC17KK9), or [#keycloak-dev](https://cloud-native.slack.com/archives/C056XU905S6) on Slack for design discussions, or questions by creating an account at [https://slack.cncf.io/](https://slack.cncf.io/).
```

Comment:
We already get a lot of people asking support questions in `#keycloak-dev` so I would rather not link it here, but in the contribution guide instead.

Reply:
@jonkoops this is in sync with what we already provide here https://www.keycloak.org/community. We should not hide any communication channels, if people are asking on the wrong channel, we need to better communicate it.

Reply:
> if people are asking on the wrong channel, we need to better communicate it.

Agreed, so let's do that here. Perhaps we seperate these items out? E.g.

```
* Join #keycloak on Slack for questions or general discussion, or join #keycloak-dev if you want to contribute and help develop Keycloak itself
```

WDYT?

Reply:
@jonkoops does it sound any better now?